activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   `TimeZone.iso8601` now accepts valid ordinal values similar to Ruby's `Date._iso8601` method.
+    A valid ordinal value will be converted to an instance of `TimeWithZone` using the `:year`
+    and `:yday` fragments returned from `Date._iso8601`.
+
+    As requested by [Andrew White](https://github.com/rails/rails/pull/41764#issuecomment-808863720)
```

Comment:
I think you can remove this line. Better to focus on how it impacts Rails users. So maybe:

```suggestion
    ```ruby
    twz = ActiveSupport::TimeZone["Eastern Time (US & Canada)"].iso8601("21087")
    twz.to_a[0, 6] == [0, 0, 0, 28, 03, 2021]
    ```
```

Reply:
@ghiculescu Thank you for the suggestion - amended.